### PR TITLE
Add speaker

### PR DESCRIPTION
**Pull request by tameeshB**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1658,6 +1658,26 @@ <h1>Speakers</h1>
                               </div>
                         </div>
 
+                          <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="M Padmal" src="img/Tameesh.jpg">
```
Comment (simsausaurabh) [inline index.html]: Use full name for image it's the convention we follow: TameeshBiswas.jpg

Comment (simsausaurabh) [inline index.html]: Write the same name in alt=" " and use use image: 300x300. Please reference the issue you are solving, if no issue raised then create an issue and reference that issue in this PR.

**Comment by simsausaurabh** [comment]
Provide preview link.

**Comment by tameeshB** [comment]
https://tameesh.in/fossasia/

**Comment by edenyay** [comment]
hi @tameeshB please try to provide description of the pull request you would like to make. Also, our developers are adding you in issue #391. If you want to use this photo, can leave comment directly there.

**Comment by tameeshB** [comment]
@edenyay Oh, I didn't see #391. I'll close this PR, thanks!

**Pull request closed without merging**